tests/php/Controllers/Root/RootControllerTest.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+use App\Http\Controllers\Root\RootController;
+use Illuminate\Http\Request;
+
+/**
+ * RootControllerクラスのテストを行う
+ */
+class RootControllerTest extends TestCase {
```

Comment:
__[phpmd]__
* The class RootControllerTest has 13 public methods. Consider refactoring RootControllerTest to keep number of public methods under 10.